Docs 2858: Document Klarna Disputes app (#1850)
MultiSafepay/docs: content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md

---
title : "Handling Klarna disputes"
meta_title: "Handling Klarna disputes - MultiSafepay Docs"
read_more: "."
weight: 1
url: '/klarna/handling-disputes/'
---
Klarna provides support for resolving order disputes with customers. For a step-by-step overview, see Klarna – [How to handle disputes at Klarna](https://docs.klarna.com/disputes/).

The easiest way to handle disputes is using the Disputes app in the Klarna Merchant Portal. The portal also gives you an overview of all your orders. 

> {{< blue-notice >}} **Note:** If viewing orders in the portal, you must still update [order and transaction statuses](/payment-methods/klarna/payment-flow/) manually in your MultiSafepay account or automatically via our API. {{< /blue-notice >}}

Reviewer: ```suggestion
{{< blue-notice >}} **Note:** If viewing orders in the Klarna Merchant Portal, you must still update [order and transaction statuses](/payment-methods/klarna/payment-flow/) manually in your MultiSafepay account or automatically via our API. {{< /blue-notice >}}
```

Reviewer: @FrancescaNCook, how are the orders automatically updated? Will Klarna inform MultiSafepay to update and then the merchant will get a notification?

Author: @Sarah-MultiSafepay  Good question, but it just means "as normal in your integration". I'll try to rephrase.

Author: ```suggestion
{{< blue-notice >}} **Note:** If viewing orders in the portal, remember you must still [change the order status to shipped](/payment-methods/klarna/changing-order-status-to-shipped/) as normal. {{< /blue-notice >}}
```

## Connecting to the Klarna Merchant Portal

1. You'll receive an email from Klarna inviting you to activate your portal account. 
    **Note:** The link expires within 3 days and is only usable once.  
2. Click **Activate account**.
3. Read the Merchant Portal Agreement, agree to the terms and conditions, and then click **Continue**.
4. Enter a password, and click **Update password**.
5. Enter your first and last names, and then click **Update profile**.
6. To sign in for the first time, enter your email address and the password you just created, and then click **Log in**.

### Setting up two-factor authentication

If you want to increase the security of your account, set up two-factor authentication with the Google Authenticator app.

1. Under **To-dos**, click **Activate two-factor authentication**. 
2. In the dialog, click **Start the setup**.
3. Open the Google Authenticator app and scan the QR code. 

> 4. Enter the one-time code from the app, and click **Set up authentication**.

Reviewer: ```suggestion
4. Enter the one-time authorization code from the app, and click **Set up authentication**.
```

## Getting started with Disputes

To open the Disputes app, in the lefthand side menu, click **Disputes**.

The side menu contains three pre-set filters to view:

- All disputes
- Unauthorized disputes

> - Diputes expiring soon

Reviewer: ```suggestion
- Disputes expiring soon
```
